lib/program-helper.js

```diff
@@ -59,13 +59,17 @@ function fileExists(cmdPath) {
         return false;
     }
 }
-
+/**
+ * @typedef {Object} HelperType
+ * @prop {() => string} getTSCCommand
+ * @prop {() => string[]} resolveTSFiles
+ * @prop {() => Command} getOptions
+ */
```

Comment:
I would *personally* favor adding this to the beginning, all the way before the "@typedef {import('commander').Command}" JSDoc comment (and in its own comment block).